Gemfile.lock

```diff
@@ -262,14 +271,15 @@ GEM
       rouge (~> 2.0.7)
     xcpretty-travis-formatter (1.0.1)
       xcpretty (~> 0.2, >= 0.0.7)
+    zeitwerk (2.5.4)
 
 PLATFORMS
+  ruby
   x86_64-darwin-20
-  x86_64-linux
 
 DEPENDENCIES
   cocoapods
   fastlane
 
 BUNDLED WITH
-   2.2.16
+   2.1.4
```

Comment:
This is a little weird. I ran `bundle install` locally, but I'm running bundler version 2.3.16... yet this PR decreases the version number. 🤔

Reply:
Don't think this matters. Will resolve it.